### ISSUE TITLE
About training loss.

### ISSUE THREAD

**Issue by Bytes-Lin** [yongliang-wu/DFT#11]
Compared with loss computed with default loss function, what will happen while training with DFT loss function in SFT training? 
The value of loss with DFT is 1000 times smaller than the loss with default loss function, is that right?

**Comment by yongliang-wu** [comment]
Based on my experiments, approximately 10-20 times is observed.

**Comment by Bytes-Lin** [comment]
> Based on my experiments, approximately 10-20 times is observed.

Thanks for replying. I'm trying on use DFT loss in an MLLM SFT task, but I observed that loss is very small. I have to make sure whether my  training code is correct.
